Decathlon/dblinter
# feat(cluster): check pg_hba.conf

# PR Description



## Link to the initial request/ticket

- Fixes #11

## What this PR Provides

- check if pg_hba is too permissive

## Known limitations / corner cases (if any)

- `N/A`

## How to test

- run a pg docker image : 
```bash
docker run -d -e POSTGRES_PASSWORD='mypass' -p 5433:5432  postgres:16  
``` 

then running dblinter should show a warning

 
### Expected results

<img width="887" alt="image" src="https://github.com/user-attachments/assets/3a7143ac-aebe-4b86-9cce-e1c3375a1d9d">
